package.json

```diff
@@ -15,8 +15,8 @@
     "npm": ">=8.0.0"
   },
   "scripts": {
-    "start-feature": "PORT=3007 NODE_ENV=development node $NODE_DEBUG_OPTION ./bin/www | bunyan -o short",
-    "start-feature:dev": "PORT=3007 NOMIS_AUTH_URL=http://localhost:9091/auth NODE_ENV=development nodemon $NODE_DEBUG_OPTION ./bin/www | bunyan -o short",
+    "start-feature": "PORT=3007 NODE_ENV=development NOTIFICATION_USERNAME=test NOTIFICATION_PASSWORD=test node $NODE_DEBUG_OPTION ./bin/www | bunyan -o short",
+    "start-feature:dev": "PORT=3007 NOMIS_AUTH_URL=http://localhost:9091/auth NODE_ENV=development NOTIFICATION_USERNAME=test NOTIFICATION_PASSWORD=test nodemon $NODE_DEBUG_OPTION ./bin/www | bunyan -o short",
```

Comment:
had to set notification area to known username/password (only for integration tests) to allow tests to pass